packages/jest/src/generators/jest-project/lib/update-jestconfig.ts

```diff
@@ -1,9 +1,26 @@
 import { JestProjectSchema } from '../schema';
 import { addPropertyToJestConfig } from '../../../utils/config/update-config';
 import { readProjectConfiguration, Tree } from '@nrwl/devkit';
+import { parse } from '@babel/parser';
+
+/**
+ * Will parse a single-statement `jest.config.js` file to see if the `projects`
+ * property exists on the exported config object.
+ *
+ * Expects a single assignment statement to an Object literal.
+ */
+function hasProjectsProperty(tree: Tree) {
+  const ast = parse(tree.read('jest.config.js').toString());
+  return !!(ast.program.body[0] as any).expression.right.properties.find(
```

Comment:
This is fancy. :sweat_smile:

Though to make it simpler, I think we can use the `jestConfigObject` to get the jest config as an object.

cc @Cammisuli